cep-XXXX.md

```diff
@@ -0,0 +1,229 @@
+- *hashed OCI-encoded <item>*: An `<item>` that is first encoded to its OCI-form and then hashed to a specific format (defined below) for use in an OCI repository `<name>` and/or `<tag>`.
+
+For further details on the OCI-defined terms, see the full OCI specifications, the [Image Spec](https://github.com/opencontainers/image-spec) and the [Distribution Spec](https://github.com/opencontainers/distribution-spec).
+
+This specification is labeled `v1` and uses only `v1.0` of the OCI specification.
+
+### conda Packages as OCI Artifacts
+
+In order to store a conda package as an OCI artifact, we need to define how the data within the conda package is mapped into the OCI blobs, the allowed blob mediaTypes, and what additional metadata (if any) is stored in the OCI manifest.
+
+#### conda OIC Artifact Blob/Layer Media Types
+
+We define the following custom media types that MUST be used for the storage of conda packages in an OCI registry.
+
+| Blob type        | Content type              | mediaType                                   |
+|------------------|---------------------------|---------------------------------------------|
+| conda package v1 | .tar.bz2 package          | application/vnd.conda.package.v1            |
+| conda package v2 | .conda package            | application/vnd.conda.package.v2            |
+| package info     | `info` folder as gzip     | application/vnd.conda.info.v1.tar+gzip      |
+| package index    | `index.json` file         | application/vnd.conda.info.index.v1+json    |
+
+#### conda OCI Artifact Blob/Layer Structure
+
+A valid conda package as an OCI artifact MUST have the following layers:
+
+- a layer holding either the `.tar.bz2` or `.conda` package file with the mediaType `application/vnd.conda.package.v1` or `application/vnd.conda.package.v2`, respectively.
+- a layer holding the `info` folder as a gzipped tarball with extension `.tar.gz` and mediaType `application/vnd.conda.info.v1.tar+gzip`.
+- a layer holding the `index.json` file as plain JSON with the mediaType `application/vnd.conda.info.index.v1+json`.
+
+Additional layers are NOT allowed in the OCI artifact.
+
+#### conda Manifest Annotations
+
+The manifest MUST have the following [Annotations](https://github.com/opencontainers/image-spec/blob/main/annotations.md#annotations):
+
+- `org.conda.oci.schema` with the value `1` indicating the `v1` specification (defined by this document).
+- `org.conda.package.name` with the name of the conda package.
+- `org.conda.package.version` with the version of the conda package.
+- `org.conda.package.build` with the build string of the conda package
+
+Additional annotations are NOT allowed.
+
+### Allowed Characters and Formats for conda Channels, Subdirs, Labels, and Package Names
+
+The follow regexes define valid conda channel names, labels, and conda package names:
```

Comment:
I assume this only standardizes names for OCI, not the whole ecosystem?

Reply:
Yes, I think these need to be put in a separate CEP (which can be added later or in parallel) where we standardize the naming schemes for all the components in the ecosystem.

Reply:
Opened https://github.com/conda/ceps/pull/116